scripts/userstyles.yml

```
@@ -336,6 +339,14 @@ userstyles:
       app-link: "https://mail.google.com"
     current-maintainers: []
     past-maintainers: [*isabelroses]
+  go-dev:
```

I know it is weird that we insist on kebab-case most of the time but we actually do allow periods in folder names!
```suggestion
  go.dev:
```
Also please change the actual directory name and associated metadata in the userstyle itself.

---

I have seen that there are other styles with periods. Just didn't know what the call was for this one. Thanks, change done.